examples/structured_2d_dgsem/elixir_euler_source_terms_sc_subcell.jl

```diff
@@ -0,0 +1,84 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+
+initial_condition = initial_condition_convergence_test
```

Comment:
This is not a test that requires IDP limiting, is it? Will this even exercise all relevant code paths then?

Reply:
True. I have the same concern. Initially, I thought this was the low density sinusoidal wave advection test ("el diablo").

Reply:
> Will this even exercise all relevant code paths then?

Hm that's a good question.
All the Newton method based limiting algorithms have a check before whether limiting is necessary or not. (So, both the limiting for non-linear variables.)
But that only affect those routines which are already used by TreeMesh. All StructuredMesh specific routines are used in all cases.
That's why everything is covered.

Nevertheless, It might be useful to adapt this elixir anyway. Or use another setup.

Reply:
I just realized that there is no subcell limiting source term elixir for TreeMesh (I thought there were). @amrueda Do you remember if we ever implemented el diablo? Or should I just use some other standard elixir ((sedov) blast, kelvin helmholtz)?

Reply:
> Smooth Isentropic Flow

Ahh, true. We didn't implement el diablo because we ended up using the Smooth Isentropic Flow test that a reviewer recommended for the MCL paper. I would use any standard elixir that requires limiting.

Reply:
In [ee86e27](https://github.com/trixi-framework/Trixi.jl/pull/1946/commits/ee86e27aa9d8c6b635445ebf8da85f082746a7b7), I added a sedov blast elixir with local limiting (global limiting in the tests).